sdk/computeschedule/.gitignore

```diff
@@ -0,0 +1 @@
+test-resources.json*
```

Comment:
This needs to be removed.   Test resources are required to be committed so that they can be used by the test pipelines for the nightly live runs.